DQM/SiPixelPhase1Heterogeneous/plugins/SiPixelPhase1CompareTrackSoA.cc

```diff
@@ -0,0 +1,200 @@
+// -*- C++ -*-
+// Package:    SiPixelPhase1CompareTrackSoA
+// Class:      SiPixelPhase1CompareTrackSoA
+//
+/**\class SiPixelPhase1CompareTrackSoA SiPixelPhase1CompareTrackSoA.cc 
+*/
+//
+// Author: Suvankar Roy Chowdhury
+//
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/Math/interface/deltaR.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+// DQM Histograming
+#include "DQMServices/Core/interface/MonitorElement.h"
+#include "DQMServices/Core/interface/DQMEDAnalyzer.h"
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "CUDADataFormats/Track/interface/PixelTrackHeterogeneous.h"
+// for string manipulations
+#include <fmt/printf.h>
+
+class SiPixelPhase1CompareTrackSoA : public DQMEDAnalyzer {
+public:
+  explicit SiPixelPhase1CompareTrackSoA(const edm::ParameterSet&);
+  ~SiPixelPhase1CompareTrackSoA() override = default;
+  void bookHistograms(DQMStore::IBooker& ibooker, edm::Run const& iRun, edm::EventSetup const& iSetup) override;
+  void analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup) override;
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
+
+private:
+  const edm::EDGetTokenT<PixelTrackHeterogeneous> tokenSoATrackCPU_;
+  const edm::EDGetTokenT<PixelTrackHeterogeneous> tokenSoATrackGPU_;
+  const std::string topFolderName_;
+  const bool useQualityCut_;
+  const pixelTrack::Quality minQuality_;
+  const float dr2cut_;
+  MonitorElement* hnTracks;
+  MonitorElement* hnLooseAndAboveTracks;
+  MonitorElement* hnLooseAndAboveTracks_matched;
+  MonitorElement* hnHits;
+  MonitorElement* hnHitsVsPhi;
+  MonitorElement* hnHitsVsEta;
+  MonitorElement* hnLayers;
+  MonitorElement* hnLayersVsPhi;
+  MonitorElement* hnLayersVsEta;
+  MonitorElement* hchi2;
+  MonitorElement* hChi2VsPhi;
+  MonitorElement* hChi2VsEta;
+  MonitorElement* hpt;
+  MonitorElement* heta;
+  MonitorElement* hphi;
+  MonitorElement* hz;
+  MonitorElement* htip;
+  MonitorElement* hquality;
```

Comment:
can you rename these to follow the usual naming convention, adding `_` at the end of their names ?